docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -13,6 +13,7 @@ The following are its source, sink, and sanitizer definitions:
 1. `@AuraEnabled`-annotated methods
 2. Any method returning a `PageReference` object
 3. `public`/`global`-scoped methods on Visualforce Controllers
+4. Methods specified through the `file#method` syntax in the `--target` parameter
```

Comment:
Since SFCA depends on SFGE and not the other way, we've been trying to keep references to SFCA outside of SFGE-specific documentation.

Can we reword this to be more generic about how individual methods are targeted? Something like:
```
4. Any method specifically targeted during invocation
```